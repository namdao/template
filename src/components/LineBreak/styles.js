import { StyleSheet } from 'react-native';
import Colors from 'constant/colors';

const styles = StyleSheet.create({
  line: {
    borderBottomColor: Colors.veryLightPink,
    borderBottomWidth: 1,
    opacity: 0.5,
  },
});

export default styles;
