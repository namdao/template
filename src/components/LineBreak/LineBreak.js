import React from 'react';
import { View, ViewPropTypes } from 'react-native';
import styles from './styles';

const LineBreak = ({ style }) => <View style={[styles.line, style]} />;

LineBreak.propTypes = {
  style: ViewPropTypes.style,
};

LineBreak.defaultProps = {
  style: {},
};

export default LineBreak;
