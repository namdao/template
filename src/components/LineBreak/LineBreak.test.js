import React from 'react';
import { shallow } from 'enzyme';
import LineBreak from './index';

const setup = (propOverrides) => {
  const props = { ...propOverrides };
  const wrapper = shallow(<LineBreak {...props} />);
  return {
    props,
    wrapper,
  };
};

describe('LineBreak check snapShot', () => {
  test('should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
