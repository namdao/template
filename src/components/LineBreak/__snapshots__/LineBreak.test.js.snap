// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineBreak check snapShot should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "#46586b",
        "borderBottomWidth": 1,
        "opacity": 0.5,
      },
      Object {},
    ]
  }
/>
`;
